packages/terra-folder-tree/src/subcomponents/FolderTreeItem.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '../clinical-lowlight-theme/FolderTree.module';
+@import '../orion-fusion-theme/FolderTree.module';
+
+:local {
+  .folder-tree-item {
+    align-items: center;
+    background-color: var(--terra-folder-tree-item-background-color, #fff);
+    border-bottom: var(--terra-folder-tree-item-border-bottom, 1px solid #dedfe0);
+    display: flex;
+    margin: 0;
+    min-height: var(--terra-folder-tree-item-min-height, 2.92857rem);
+    overflow-wrap: anywhere;
```

Comment:
We would want this to be "break-word".  I don't think we want to use anywhere as our option.

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
With `break-word`, long strings without spaces do not wrap which is why I used `anywhere`, do you know of a way to handle that with `break-word`?

<img width="478" alt="image" src="https://github.com/cerner/terra-framework/assets/42594047/8181a440-4470-49a1-8baa-3d2e3ba58059">